components/core/src/clp_s/search/StringLiteral.hpp

```diff
@@ -69,18 +70,8 @@ class StringLiteral : public Literal {
         }
 
         // If '?' and '*' are not escaped, we add LiteralType::ClpStringT to m_string_type
```

Comment:
I am personally fine with having this comment since I love reading, lol
But since we were talking about comments, what's your opinion?

Reply:
When I read the code, I get the same message as the comment (without a loss of speed), so I would drop this comment. Also, this is a minor in this case, but it's possible one day we introduce a new wildcard, making this comment inaccurate (and it's hard to reliably track down these types of inaccurate comments).